src/components/library/SearchLyrics.vue

```diff
@@ -170,10 +170,12 @@ const initialize = async () => {
   title.value = props.searchingTrack.title
   albumName.value = props.searchingTrack.album_name
   artistName.value = props.searchingTrack.artist_name
+
+  await doSearchLyrics()
 }
 
 onMounted(async () => {
   await initialize()
-  doSearchLyrics()
+doSearchLyrics()
```

Comment:
I think this change is unnecessary and results in incorrect indentation

```suggestion
  doSearchLyrics()
```